packages/synapse-react-client/src/theme/DefaultTheme.tsx

```diff
@@ -112,6 +112,13 @@ export const defaultMuiThemeOptions: ThemeOptions = {
                 : theme.palette[ownerState.color || 'primary']['main'],
             textDecorationThickness: '2px',
           },
+          '&:hover, &:visited': {
+            // Overrides a base style in Bootstrap 3 and in SWC
+            color:
+              ownerState.color === 'inherit'
+                ? ownerState.color
+                : theme.palette[ownerState.color || 'primary']['main'],
+          },
```

Comment:
Ensure that text buttons that are rendered as links have the correct color

before: 

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/05738979-829b-474a-bafe-9ceef73b40cd

after:

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/d0f7f801-42e3-46db-a401-5db17fd12da2